src/citrine/resources/team.py

```diff
@@ -99,37 +101,81 @@ def list_members(self) -> List[TeamMember]:
                                             version=self._api_version)["users"]
         return [TeamMember(user=User.build(m), team=self, actions=m["actions"]) for m in members]
 
-    def remove_user(self, user_id: Union[str, UUID]) -> bool:
+    def get_member(self, user_id: Union[str, UUID, User]) -> TeamMember:
         """
-        Remove a User from a Team.
+        Get a particular member in the current team.
+
+        May require admin privileges depending on which user is being requested.
 
         Parameters
         ----------
         user_id: str or uuid
             The id of the user to remove from the team
 
+        Returns
+        -------
+        TeamMember
+            The requested team member
+
+        """
+        if isinstance(user_id, User):
+            user_id = user_id.uid
+        path = self._path() + format_escaped_url('/users/{user_id}', user_id=user_id)
+        member = self.session.get_resource(path=path, version=self._api_version)["user"]
```

Comment:
Won't this raise a `KeyError` on a 404 (or other error) and obscure their mistake as a result?

Reply:
citrine-python raises a Unauthorized exception earlier in the stack if you don't get a 200.  The only case where this misbehaves should if the platform returned a 200 but the JSON didn't conform to the API.  If this is a concern, I'm happy to add additional care (though this follows the pattern established in `list_members`).